keyboards/ergodash/ergodash.h

```diff
@@ -0,0 +1,29 @@
+#ifndef ERGODASH_H
+#define ERGODASH_H
+
+#include "quantum.h"
+
+#ifdef KEYBOARD_ergodash_rev1
+    #include "rev1.h"
+#endif
+
+
+// Used to create a keymap using only KC_ prefixed keys
+#define KC_KEYMAP( \
```

Comment:
Could you switch the `KC_KEYMAP` to `LAYOUT_kc`?

Reply:
Yes, I will.